test/smoke/LidoARMSmokeTest.t.sol

```diff
@@ -204,4 +204,30 @@ contract Fork_LidoARM_Smoke_Test is AbstractSmokeTest {
         vm.prank(operator);
         lidoARM.setOperator(operator);
     }
+
+    error InvalidInitialization();
+
+    // Can not be called again after reinitialized by the deploy script
+    function test_registerLidoWithdrawalRequests() external {
+        vm.expectRevert(InvalidInitialization.selector);
+        vm.prank(operator);
+        lidoARM.registerLidoWithdrawalRequests();
+    }
+
+    function test_lidoWithdrawalRequests() external {
+        uint256 totalAmountRequested = 0;
+        uint256[] memory requestIds = IStETHWithdrawal(Mainnet.LIDO_WITHDRAWAL).getWithdrawalRequests(address(lidoARM));
+        // Get the status of all the withdrawal requests. eg amount, owner, claimed status
+        IStETHWithdrawal.WithdrawalRequestStatus[] memory statuses =
+            IStETHWithdrawal(Mainnet.LIDO_WITHDRAWAL).getWithdrawalStatus(requestIds);
+
+        console.log("Got %d withdrawal requests", requestIds.length);
+
+        for (uint256 i = 0; i < requestIds.length; i++) {
+            assertEq(lidoARM.lidoWithdrawalRequests(requestIds[i]), statuses[i].amountOfStETH);
+            totalAmountRequested += statuses[i].amountOfStETH;
+        }
+
+        assertEq(totalAmountRequested, lidoARM.lidoWithdrawalQueueAmount());
+    }
```

Comment:
I locally upgunned these tests and verified that at the right historical block, we could do all withdraws cleanly, and lidoWithdrawalQueueAmount went to zero.